lib/rules/web/admin_console/4.8/deployment.xyaml

```diff
@@ -149,4 +149,38 @@ select_replica_count_and_selector:
   params:
     text: Replica count and selector
   action: click_checkbox_input_by_text
-
+check_and_click_metrics_tab:
+  elements:
+  - selector:
+      xpath: //*[@id="content-scrollable"]//a[contains(text(),'Metrics')]
+  params:
+    tab_name: Metrics
+  action: click_tab
+check_charts_on_metrics_tab:
+  elements:
+  - selector:
+      xpath: //h2[@class='co-dashboard-card__title' and text()='Memory usage']
+    timeout: 10
+  - selector:
+      xpath: //h2[@class='co-dashboard-card__title' and text()='CPU usage']
+    timeout: 10
+  - selector:
+      xpath: //h2[@class='co-dashboard-card__title' and text()='Filesystem']
```

Comment:
better use `contains` 
```
//h2[contains(@class,'co-dashboard-card__title') and contains(text(),'Filesystem')]
```

Reply:
The last but most important, we need to make sure `No datapoints found` should not be shown for workloads already have traffic